sdk/storage/Azure.Storage.DataMovement/tests/Shared/StartTransferUploadTestBase.cs

```diff
@@ -754,6 +754,6 @@ public async Task LocalToRemoteObject_LargeMultiple(int objectCount, long size,
                 waitTimeInSec: waitTimeInSec,
                 objectCount: objectCount);
         }
-#endregion
+        #endregion
```

Comment:
nit: I think this is not supposed to be indented?

Reply:
It was actually supposed to be indented. Cause the region it's encompassing is all indented.